Python-packages/covidcast-py/covidcast/covidcast.py

```diff
@@ -175,26 +175,8 @@ def signal(data_source: str,
                          "start_day = '{start}', end_day = '{end}'".format(
                              start=start_day, end=end_day))
 
-    if isinstance(geo_values, str):
-        # User only provided one, not a list
-        geo_values = [geo_values]
-
-    dfs = [
-        _fetch_single_geo(
-            data_source, signal, start_day, end_day, geo_type, geo_value,
-            as_of, issues, lag)
-        for geo_value in set(geo_values)
-    ]
-
-    try:
-        # pd.concat automatically filters out None
-        out = pd.concat(dfs)
-    except ValueError:
-        # pd.concat raises ValueError if all of the dfs are None, meaning we
-        # found no data
-        return None
-
-    return out
+    return _fetch_single_geo(
+        data_source, signal, start_day, end_day, geo_type, geo_values, as_of, issues, lag)
```

Comment:
seems like we should merge this function and `_fetch_single_geo`, since `_fetch_single_geo` no longer fetches a single geo?

Reply:
Renamed to _fetch_epidata and refactored a little bit. Didn't merge  it with signal() yet in case we go ahead with #429